src/NUnitFramework/framework/Internal/Commands/TestMethodCommand.cs

```diff
@@ -89,7 +92,21 @@ private object RunTestMethod(TestExecutionContext context)
 
         private object InvokeTestMethod(TestExecutionContext context)
         {
-            return testMethod.Method.Invoke(context.TestObject, arguments);
+		    var methodInfo = MethodInfoCache.Get(_testMethod.Method);
```

Comment:
nitpick: Very minor whitespace issue, I think there's two tabs at the beginning of the line throwing this off.
Given this uses `MethodInfoCache` this likely accidentally came into the recent rebasing

```suggestion
            var methodInfo = MethodInfoCache.Get(_testMethod.Method);
```